lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1337,6 +1389,40 @@ def grpc_severity(severity)
       severity
     end
 
+    def parse_int(v)
```

Comment:
We normally try to avoid one-letter variable names if possible. So can we do this instead? And as mentioned in the other comment, let's add a `parse_string` method as well.

```
    def parse_string(value)
      value.to_s
    end

    def parse_int(value)
      value.to_i
    end

    def parse_bool(v)
      [true, 'true', 1].include?(value)
    end
```